include/seqan3/alphabet/all.hpp

```diff
@@ -119,9 +107,36 @@
  *
  * Note, however, that literals **are not** required by the concept.
  *
- * <small>In the documentation you will also encounter seqan3::Semialphabet. It describes "one half" of an
- * alphabet and only defines the rank interface as a type requirement. It is mainly used internally and not
- * relevant to most users of SeqAn.</small>
+ * ### Different concepts
+ *
+ * All types that have valid implementations of the functions/functors described above model the concept
+ * seqan3::WritableAlphabet. This is the strongest (i.e. most refined) *general case* concept.
+ * There are more refined concepts for specific biological applications (like seqan3::NucleotideAlphabet), and there are
+ * less refined concepts that only model part of an alphabet:
+ *
+ *   * seqan3::Semialphabet and derived concepts only require the rank interface;
+ *   * seqan3::Alphabet (without `Writable*`) and derived concepts only require readability and not assignability.
+ *
+ * Typically you will use seqan3::Alphabet in "read-only" situations (e.g. `const` parameters) and
+ * seqan3::WritableAlphabet whenever the values might be changed.
+ * Semi-alphabets are less useful in application code.
+ *
+ * |                                  | seqan3::Semialphabet | seqan3::WritableSemialphabet | seqan3::Alphabet | seqan3::WritableAlphabet | Aux |
+ * |----------------------------------|:--------------------:|:----------------------------:|:----------------:|:------------------------:|:---:|
+ * | seqan3::alphabet_size_v          | ✅                    | ✅                            | ✅                | ✅                        |     |
+ * | seqan3::to_rank                  | ✅                    | ✅                            | ✅                | ✅                        |     |
+ * | seqan3::alphabet_rank_t          | ✅                    | ✅                            | ✅                | ✅                        |  🔗 |
+ * | seqan3::assign_rank_to           |                      | ✅                            |                  | ✅                        |     |
```

Comment:
can you please align all the column borders

Reply:
They are aligned. It's just that the UTF8-characters are not part of your monospaced font and therefore the rendering off.